content/streaming-api/index.textile

```diff
@@ -0,0 +1,115 @@
+
+h2(#authentication). Authentication
+
+The same authentication options as used in the REST api also apply here. So for basic auth, a querystring parameter of @key@ or an @Authorization: Basic <base64-encoded key>@ header; for token auth, a querystring parameter of @accessToken@ containing an access token or an @Authorization: Bearer <base64-encoded token>@ header. See "REST API authentication":/rest-api#authentication for more information.
+
+
+h2(#connection-state-recovery). Connection state recovery
+
+If you are disconnected, you can supply the @id@ of the last event you received as the @lastEvent@ querystring parameter of a new connection, to resume from that point.
+
+Connection state is only retained for two minutes. See "Connection state explained":/realtime/connection#connection-state-explained for full documentation.
+
+Due to this feature being in beta, it is currently subject to other restrictions. For example, at the moment you can not resume from part-way through a backlog of messages that was being sent as part of a previous resume; you can only resume from a message (so you can't resume unless you've received at least one message on the connection); and you can only resume from the id of a message that was sent in the last two minutes (so it will not work if the last message sent on the connection was more than two minutes ago). We expect to fix all of these by the time this feature graduates out of beta.
+
+h1(#api). API routes
+
+h3(#event-stream). Plain event stream
+
+h6. GET realtime.ably.io/event-stream
+
+Start a streaming HTTP request.
+
+h5. Request parameters
+
+- channels := (mandatory) one or more channel names, separated by commas (or the @separator@ if specified). Non-url-safe characters should be URL-encoded (for example, @?channels=foo%3Fbar@ will subscribe to the channel @foo?bar@). Alias: @channel@
+- separator := (optional) a separator, to enable easy subscriptions to channels with commas in their name. For example, @?channels=?separator=|&channel=fo,o|ba,r@ will subcribe to the two channels @fo,o@ and @ba,r@.
+- v := (mandatory) the version of the api you are requesting. The current (and only) version of the API is 1.1, so only @v=1.1@ is accepted
+- key := (optional) an Ably API key to use, if using basic auth
+- accessToken := (optional) an Ably auth token to use, if using token auth
+- lastEvent := (optional) an @id@ to resume from, see "connection state recovery":#recovery below
+- enveloped := (optional, default true) if true, the @data@ from each event envelope for a @message@ event will be a "Message":/realtime/types#message object. If false, it will be the payload from the message directly. See "Envelope format":#envelope-format below
+- heartbeats := (optional, default false) if @true@ will use an explicit heartbeat event rather than a newline as a keepalive packet
```

Comment:
I thought we were going to support fields so that subscribers could select only the fields they want sent.

Reply:
Yeah, I just didn't do it in the MVP (wasn't essential).  I'll add it at some point, maybe as part of the work I do next week